packages/walletconnect/package.json

```diff
@@ -1,30 +1,21 @@
 {
   "author": "thorswap",
   "dependencies": {
-    "@thorswap-lib/helpers": "workspace:*",
-    "@thorswap-lib/types": "workspace:*",
-    "@walletconnect/client": "1.8.0",
-    "@walletconnect/qrcode-modal": "1.8.0",
-    "@walletconnect/web3-provider": "1.8.0",
-    "base64-js": "1.5.1",
-    "bech32": "2.0.0"
+    "@ethersproject/bytes": "^5.7.0",
+    "@ethersproject/strings": "^5.7.0",
+    "@thorswap-lib/toolbox-evm": "workspace:^",
```

Comment:
This should be in dev & peer dependencies with being marked as external module in `vite.config.js` as those will be installed with `toolbox-evm`